tests/sentry/tasks/test_assemble.py

```diff
@@ -191,7 +196,45 @@ class AssembleArtifactsTest(BaseAssembleTest):
     def setUp(self):
         super().setUp()
 
-    def test_artifacts(self):
+    def test_artifacts_with_debug_ids(self):
+        bundle_file = self.create_artifact_bundle(
+            fixture_path="artifact_bundle_debug_ids", project=self.project.id
+        )
+        blob1 = FileBlob.from_file(ContentFile(bundle_file))
+        total_checksum = sha1(bundle_file).hexdigest()
+        debug_ids = ["eb6e60f1-65ff-4f6f-adff-f1bbeded627b"]
+
+        assemble_artifacts(
+            org_id=self.organization.id,
+            project_id=self.project.id,
+            version=None,
+            checksum=total_checksum,
+            chunks=[blob1.checksum],
+        )
+
+        assert self.release.count_artifacts() == 0
+
+        status, details = get_assemble_status(
+            AssembleTask.ARTIFACTS, self.organization.id, total_checksum
+        )
+        assert status == ChunkFileState.OK
+        assert details is None
+
+        for debug_id in debug_ids:
+            debug_id_artifact_bundles = DebugIdArtifactBundle.objects.filter(debug_id=debug_id)
```

Comment:
It would be dangerous to write code like this outside of tests becauase it can leak cross organization data (eg: same debug id existing in more than one org). Before someone copy pastes this elsewhere, please make sure that this has the `organization_id` as constraint too.

Reply:
Oh yeah, thanks for the heads-up! I forgot about that